doc/user/content/sql/create-sink.md

```diff
@@ -53,7 +53,7 @@ Field | Value type | Description
 #### SSL `WITH` options
 
 Use the following options to connect Materialize to an SSL-encrypted Kafka
-cluster.
+cluster. For more detail, see [SSL-encrypted Kafka details](#ssl-encrypted-kafka-details).
```

Comment:
This link currently only exists at `/sql/create-source/avro-kafka/#ssl-encrypted-kafka-details`